spec.md

```diff
@@ -491,6 +499,18 @@ The foundation block simply sequences operations and other blocks
 }
 ```
 
+### Parallel quantum block
+
+A grouping of quantum operations to be performed in parallel
+
+```json5
+{
+  "block": "parallel",
+  "ops": [{...}, ...],
+  "metadata": {...}  // Optional
+}
+```
```

Comment:
We should add a note here that only quantum operations are allowed in the `"ops"` field here.

I'd also like to see whether @qciaran thinks the block name should be called "qparallel" or some such name to be clear that this is only relevant for quantum operations.

Reply:
@Asa-Kosto-QTM unsure if this was missed. We restrict the ops to be `QOp` in the model, so the spec should clearly say that. General `"ops"` allow blocks and all kinds of ops within them.